src/lib.cairo

```diff
@@ -4,6 +4,14 @@ fn main() {
     println!("check_sum fn result here: {result}");
     let sum: u16 = add_num(10, 5);
     println!("the sum of 2 numbers is: {sum}");
+    let sum: u16 = add_num(5, 3);
+    println!("The sum of 5 and 3 equal {}", sum);
+    let sub: u16 = sub_num(2, 1);
+    println!("The subtraction of 2 and 1 equal {}", sub);
+    let mul_val: u16 = mul_num(2, 2);
+    println!("The multiplication of 2 and 2 equal {}", mul_val);
+    let div_val: u16 = div_num(10, 0);
+    println!("The division of 10 and 2 equal {}", div_val);
```

Comment:
Great job with the assignment. 
your `div_num` logic is passing; 
however, you don't have to hardcode the the values like so:
`The division of 10 and 2 equal {}`

when 10 and 5 are passed into `div_num` function like so:
`div_num(10, 0);`
it  prints it thus: `The division of 10 and 2 equal 2`
Meaning, it is loggin incorrectly

Although not compulsory, if you have to `println!`, make you it is clear enough so that anyone can understand what it is you're trying to log
`println!("The quotient of 2 numbers is {}", div_val);`

Reply:
same thing applies to all the other functions you added
Kudos!

Reply:
Noted on it